container_linux.go

```diff
@@ -135,6 +135,7 @@ func (c *linuxContainer) commandTemplate(p *Process, childPipe *os.File) (*exec.
 	cmd.Stdin = p.Stdin
 	cmd.Stdout = p.Stdout
 	cmd.Stderr = p.Stderr
+	cmd.ExtraFiles = p.ExtraFiles
```

Comment:
The correct code would look like:

`cmd.ExtraFiles = append(p.ExtraFiles, childPipe)`